client/lib/user-settings/index.js

```diff
@@ -14,6 +16,19 @@ var Emitter = require( 'lib/mixins/emitter' ),
 	user = require( 'lib/user' )();
 
 /**
+ * Decodes entities in those specific user settings properties
+ * that the REST API returns already HTML-encoded
+ */
+function decodeUserSettingsEntities( data ) {
+	let decodedValues = {
+			display_name: data.display_name && decodeEntities( data.display_name ),
+			description: data.description && decodeEntities( data.description ),
+			user_URL: data.user_URL && decodeEntities( data.user_URL )
```

Comment:
Indentation is off in this object. There should be one less tab for each property.